### PR TITLE
Enable Kovan bridge liquidity

### PR DESCRIPTION
**Pull request by fragosti**
I'm seeing `WETH`, `MKR`, `REP`, `ZRX`, `BAT`, `SAI` and maybe more liquidity.
I'm not seeing any liquidity for `USDC`. I don't think `USDC` is really used that much (?) but the lack of `DAI` is concerning. 

For `buyToken=DAI` queries I get an "out of Gas" error (not seen here). For `sellToken=DAI` queries it works.
```
Error: VM execution error.
    at SwapHandlers.<anonymous> (/Users/francescoagosti/Projects/0x-api/src/handlers/swap_handlers.ts:82:23)
    at Generator.throw (<anonymous>)
    at rejected (/Users/francescoagosti/Projects/0x-api/src/handlers/swap_handlers.ts:5:65)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

### Update

By disabling kyber we no longer get the DAI errors.

### CODE REVIEW

File: src/token_metadatas_for_networks.ts [hunk 1]
```diff
@@ -81,7 +82,7 @@ export const TokenMetadatasForChains: TokenMetadataAndChainAddresses[] = [
         decimals: 18,
         tokenAddresses: {
             [ChainId.Mainnet]: '0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2',
-            [ChainId.Kovan]: NULL_ADDRESS,
+            [ChainId.Kovan]: '0xac94ea989f6955c67200dd67f0101e1865a560ea',
```
Comment (dorothy-zbornak) [inline src/token_metadatas_for_networks.ts]: 0xAaF64BFCC32d0F15873a02163e7E500671a4ffcD ?

Comment (dorothy-zbornak) [reply src/token_metadatas_for_networks.ts]: For context: https://developer.kyber.network/docs/Environments-Kovan/

File: src/token_metadatas_for_networks.ts [hunk 2]
```diff
@@ -71,7 +72,7 @@ export const TokenMetadatasForChains: TokenMetadataAndChainAddresses[] = [
         decimals: 18,
         tokenAddresses: {
             [ChainId.Mainnet]: '0x0d8775f648430679a709e98d2b0cb6250d2887ef',
-            [ChainId.Kovan]: NULL_ADDRESS,
+            [ChainId.Kovan]: '0x9ddb308c14f700d397bb26f584ac2e303cdc7365',
```
Comment (dekz) [inline src/token_metadatas_for_networks.ts]: https://github.com/makerdao/cage-keeper/blob/master/config/kovan-addresses-0_2_17_2.json#L35

Has BAT as `0x9f8cfb61d3b2af62864408dd703f9c3beb55dff7`. WHICH ONE IS OFFICIAL???

File: src/token_metadatas_for_networks.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export interface TokenMetadataAndChainAddresses {
 }
 
 // Most token metadata taken from https://github.com/MetaMask/eth-contract-metadata/
+// And https://github.com/compound-finance/compound-protocol/blob/master/networks/kovan.json
 // tslint:disable:max-file-line-count
 export const TokenMetadatasForChains: TokenMetadataAndChainAddresses[] = [
     {
```
Comment (dekz) [inline src/token_metadatas_for_networks.ts]: Can't click on DAI address, but this one might be SAI for Kovan.

https://kovan.etherscan.io/address/0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa 

https://github.com/makerdao/developerguides/tree/master/remittance#live-dai-token-smart-contract-deployments

Comment (fragosti) [reply src/token_metadatas_for_networks.ts]: I get a `VM execution error` when using this one.

File: src/token_metadatas_for_networks.ts (hunk 1 above)
Comment (fragosti) [reply src/token_metadatas_for_networks.ts]: This one works too and is from Kyber so going with this one, thanks!

File: src/token_metadatas_for_networks.ts (hunk 2 above)
Comment (fragosti) [reply src/token_metadatas_for_networks.ts]: This one worked!

**Pull request merged**